plugins/gcloud/lib/samson_gcloud/image_builder.rb

```diff
@@ -59,6 +60,12 @@ def build_image(dir, build, executor, tag_as_latest:, cache_from:)
 
       private
 
+      def gcloud_new?
```

Comment:
would prefer if this method was gcloud_version and you did the comparison where you use it...

Reply:
(new is relative)

Reply:
yeah gcloud_version was also nice because it removes the `false` caching logic which is always annoying